### ISSUE TITLE
`consistent-existence-index-check` add option greater-than and none-zero

### ISSUE THREAD

**Issue by ximex** [sindresorhus/eslint-plugin-unicorn#2471]
### Description

I want to have the same option as `explicit-length-check` to choose between `greater-than` and `none-zero`.

`none-zero` could stay as default but if changed to `greater-than` it should behave as below

### Fail

```js
const index = foo.indexOf('bar');
if (index !== -1) {}
```

### Pass

```js
const index = foo.indexOf('bar');
if (index > -1) {}
```

### Additional Info

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/explicit-length-check.md#options

**Comment by fisker** [comment]
Any reason you don't want `=== -1` and `!== -1`?

**Comment by ximex** [comment]
there is no hard reason. i know that it makes no different but it is more precise as value couldn't be below `-1`. and as the other rule has also the option i think this one could also get it

**Comment by axetroy** [comment]
Use `-1` to make the style be kept consistent

Otherwise, someone would write code like this

`index > 0`
`index > -1`
`index < 0`
`index <= 0`

This makes the code confusing

**Comment by sindresorhus** [comment]
I'm going to pass on this. I don't see a strong enough reason to add an option for this.
